config/routes.rb

```diff
@@ -4,9 +4,9 @@
   resources :orders
   resources :orders_blogs
 
-  root 'orders#index'
+root 'orders#index'
 
-  devise_for :drivers, skip: :all
+devise_for :drivers, skip: :all, controllers: { sessions: 'drivers/sessions' }
   as :driver do
     get 'driver' => 'devise/sessions#new', :as => :new_driver_session
```

Comment:
Please use the Ruby 1.9 hash literal syntax when your hash keys are symbols. 
https://github.com/bbatsov/ruby-style-guide#hash-literals